<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>409</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.選擇檔案(excel)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_excel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>開啟檔案</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.選擇分頁</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_select_tab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.輸出位置(目錄)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output_dir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_output_dir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>選擇目錄</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_waveform">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>480</width>
     <height>120</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>480</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_No">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rec">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>REC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_excel_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.顯示欄位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4.檔名欄位</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_show">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_filename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
